core/src/test/java/org/testcontainers/images/RemoteDockerImageTest.java

```diff
@@ -0,0 +1,47 @@
+package org.testcontainers.images;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.testcontainers.utility.Base58;
+
+import java.util.concurrent.CompletableFuture;
+
+public class RemoteDockerImageTest {
+
+    @Test
+    public void toStringContainsOnlyImageName() {
+        String imageName = Base58.randomString(8).toLowerCase();
```

Comment:
Why random string?

Reply:
why not? :)

Reply:
Because constant example string seems more controlled to me.